plugins/woocommerce/client/legacy/css/admin.scss

```diff
@@ -4160,6 +4160,7 @@ table.wc_shipping {
 }
 
 .wc-shipping-zone-settings {
+	margin-top: 8px !important;
```

Comment:
I usually try to avoid `!important` so just flagging it here if there is a better way to do it, if not, then we can go with it.

Reply:
I had the same feeling, usually we can get around it by being more specific when targeting the selector or there's some other rule somewhere which is overriding it

Reply:
I suggest using the `$gap-smaller` instead.